### PR TITLE
Adding command descriptions

### LINKED ISSUE #1108: Add --help description of commands
Some commands don't have a description on --help, check-update, bundle-add, bundle-list, bundle-remove, mirror, update.

### PR DESCRIPTION
**Pull request by castulo**
This commit adds a shoort description for the help menu of all swupd
commands.

Closes #1108 

Signed-off-by: Castulo Martinez <castulo.martinez@intel.com>

### CODE REVIEW

File: src/check_update.c [hunk 1]
```diff
@@ -34,6 +34,7 @@
 
 static void print_help(void)
 {
+	print("Checks whether an update is available and prints out the information if so\n\n");
```
Comment (karthikprabhuvinod) [inline src/check_update.c]: I would quote this as

Checks whether an update is available along with current & server version

**Comment by karthikprabhuvinod** [comment]
@castulo : Hey Castulo , this is a good addition. I was thinking we should have something like example section for all swupd commands which lists a few popular ways of using it.... helps a beginner ramp up fast if he/she comes from another linux distro world?

for eg for update: 
**Usage:**

**Description:**

**Example:**
swupd update -V 10  -- update to a specific version
swupd update            -- updates to latest version 

Maybe we can discuss this once @otaviobp is back. Let me know what you feel?

**Comment by castulo** [comment]
> @castulo : Hey Castulo , this is a good addition. I was thinking we should also have something like example section for all swupd commands which lists a few popular ways of using it.... helps a beginner ramp up fast if he/she comes from another linux distro world?
> 
> for eg for update:
> **Usage:**
> 
> **Description:**
> 
> **Example:**
> swupd update -V 10 -- update to a specific version
> swupd update -- updates to latest version
> 
> Maybe we can discuss this once @otaviobp is back. Let me know what you feel?

👍 yeah we can discuss it with @otaviobp, I like the idea of having examples but I'm not sure the "--help" menus are the best place for them or if we should just have them in the manual. In either case that is separate from this PR.

File: src/check_update.c (hunk 1 above)
Comment (castulo) [reply src/check_update.c]: Well, to check if there is an update available it needs to check the current version and the server version. So it kind of seems redundant to me that way.

Comment (karthikprabhuvinod) [reply src/check_update.c]: hmm thinking of it makes sense

**Comment by karthikprabhuvinod** [comment]
@castulo : Suggest to add a print inside print_help for third_party command like

print("3rd-party provides users a mechanism to build, add custom packages which are not present upstream \n\n");

Because if I am new to swupd and type sudo ./swupd 3rd-party ...there is no way for me to really understand what the term actually means

**Comment by castulo** [comment]
> @castulo : Suggest to add a print inside print_help for third_party command like
> 

Good catch @karthikprabhuvinod , I forgot that general command. I'll add a short description to it.

**Comment by castulo** [comment]
Moved this commit to PR #1260 so I'm closing this PR.

**Pull request closed without merging**